client/checkout/classic/sepa.js

```diff
@@ -0,0 +1,385 @@
+/* global jQuery */
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { PAYMENT_METHOD_NAME } from '../constants.js';
+import { getConfig } from 'utils/checkout';
+import WCPayAPI from './../api';
+import enqueueFraudScripts from 'fraud-scripts';
+
+jQuery( function ( $ ) {
+	enqueueFraudScripts( getConfig( 'fraudServices' ) );
+
+	const publishableKey = getConfig( 'publishableKey' );
+
+	if ( ! publishableKey ) {
+		// If no configuration is present, probably this is not the checkout page.
+		return;
+	}
+
+	// Create an API object, which will be used throughout the checkout.
+	const api = new WCPayAPI(
+		{
+			publishableKey,
+			accountId: getConfig( 'accountId' ),
+		},
+		// A promise-based interface to jQuery.post.
+		( url, args ) => {
+			return new Promise( ( resolve, reject ) => {
+				jQuery.post( url, args ).then( resolve ).fail( reject );
+			} );
+		}
+	);
+	const elements = api.getStripe().elements();
+
+	// In the future this object will be loaded with customer information through `wp_localize_script`.
+	const preparedCustomerData = {};
+
+	// Create a card element.
+	const cardElement = elements.create( 'iban', {
+		supportedCountries: ["SEPA"],
+		hidePostalCode: true,
+		classes: { base: 'wcpay-sepa-card-mounted' },
+	} );
+    console.log(cardElement);
+
+	// Only attempt to mount the card element once that section of the page has loaded. We can use the updated_checkout
+	// event for this. This part of the page can also reload based on changes to checkout details, so we call unmount
+	// first to ensure the card element is re-mounted correctly.
+	$( document.body ).on( 'updated_checkout', () => {
+		// If the card element selector doesn't exist, then do nothing (for example, when a 100% discount coupon is applied).
+		// We also don't re-mount if already mounted in DOM.
+		if (
+			! $( '#wcpay-sepa-card-element' ).length ||
```

Comment:
Created a new div called `wcpay-sepa-card-element` so i needed to replace `wcpay-card-element` to `wcpay-sepa-card-element`